server.js

```diff
@@ -22,6 +24,27 @@ io.on('connection', (socket) => {
 
     socket.broadcast.emit(EVENTS.LISTEN_ENTER_CHANNEL, userData);
   });
+
+  socket.on('new chat', ({ channelId, newChat }) => {
+    socket.join(channelId);
+
+    if (!chatContents[channelId]) {
+      const data = {
+        channelId,
+        chatList: [{ author: newChat.author, chat: newChat.chat }],
```

Comment:
정확한 구조는 잘 모르지만, newChat이 author와 chat속성을 가지고 있는 객체라면, 

```suggestion
        chatList: [newChat],
```
이렇게만 해도 되지 않을까요?

Reply:
헛.. 수정하겠습니다